archive/readme.md

```diff
@@ -0,0 +1,6 @@
+# Archive
+
+Here's anything we've done before and interated on:
+
+## Agreements
+ - [Financial V01](archive/financial-agreement-v01.md)
```

Comment:
i didn't give it that much thought - should probably be  1.0.1 as thats what it is tagged as